view/frontend/templates/checkout/success.phtml

```diff
@@ -25,7 +25,32 @@
  * @var \Adyen\Payment\Block\Checkout\Success $block
  */
 ?>
-<?php if ($block->isBoletoPayment()): ?>
+
+
+<?php if ($block->renderAction()): ?>
+    <script type="text/javascript">
```

Comment:
is there a way to outsource it into a separate JS file?

Reply:
No as it needs the php tags for the vraiables.